test/benchmark.cc

```diff
@@ -845,6 +1084,7 @@ int main(int args, char** argv){
 	fprintf(stdout, "Type: uint32_t\n");
 	fprintf(stdout, "============================================================\n\n");
 
+#if 0
```

Comment:
Dead code?

Reply:
No, the `#if 0` code blocks were added to skip all but the stack benchmark tests. I have removed the `#if 0` code.